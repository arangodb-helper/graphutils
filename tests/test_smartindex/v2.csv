_key|Hello|smart
111|1|C
333|3|D
