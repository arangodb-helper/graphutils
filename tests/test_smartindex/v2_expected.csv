_key|Hello|smart
11:111|1|11
33:333|3|33
